src/main/resources/db/changelog/20201103112757-purge-current-sca-certs.xml

```diff
@@ -20,8 +20,29 @@
             <column name="cont_acc_cert_id"/> <!-- this sets the column value to null -->
         </update>
 
-        <delete tableName="cp_cont_access_cert" />
+        <dropForeignKeyConstraint baseTableName="cp_consumer" constraintName="fk_cont_acc_cert" />
+
+        <sql dbms="mysql,mariadb">
```

Comment:
For operations that liquibase actually supports properly, we should use its tags (i.e. the delete tag that this update is replacing).

However, this block is unnecessary with changes to the SQL block below.

Reply:
This was intented to keep MySQL/MariaDB away from Truncate with Cascade. But now I have removed the cascade from both tables. Hence removed this SQL tag as well.